docs/seccompiler.md

```diff
@@ -2,28 +2,28 @@
 
 ## Overview
 
-Seccompiler is a tool that compiles seccomp filters expressed as JSON files
+Seccompiler-bin is a tool that compiles seccomp filters expressed as JSON files
 into serialized, binary BPF code that is directly consumed by Firecracker,
 at build or launch time.
 
-Seccompiler defines a custom [JSON file structure](#json-file-format), detailed
-further below, that the filters must adhere to.
+Seccompiler-bin uses a custom [JSON file structure](#json-file-format),
+detailed further below, that the filters must adhere to.
 
-Besides the compiler binary, seccompiler also exports a small library
-interface, with a couple of helper functions, for deserializing and installing
-the binary filters.
+Besides the seccompiler-bin executable, seccompiler also exports a small
+library interface, with a couple of helper functions, for deserializing and
```

Comment:
seccompiler also exports a library interface, with helper functions for deserializing......

Reply:
Done